2025-05-27 00:00:25 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_lstm
2025-05-27 00:07:16 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_lstm
2025-05-27 00:15:39 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_lstm
2025-05-27 00:28:31 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_lstm
2025-05-27 01:21:48 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1h_lstm
2025-05-27 01:28:48 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1d_lstm
2025-05-27 01:32:57 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_lstm
2025-05-27 01:33:11 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-lstm: "['target'] not found in axis"
2025-05-27 01:33:11 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_gru
2025-05-27 01:33:25 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-gru: "['target'] not found in axis"
2025-05-27 01:33:25 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_transformer
2025-05-27 01:33:40 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-transformer: "['target'] not found in axis"
2025-05-27 01:33:40 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1w_lstm
2025-05-27 01:33:54 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1w-lstm: "['target'] not found in axis"
2025-05-27 01:33:54 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1w_gru
2025-05-27 01:34:09 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1w-gru: "['target'] not found in axis"
2025-05-27 01:34:09 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1w_transformer
2025-05-27 01:34:23 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1w-transformer: "['target'] not found in axis"
2025-05-27 01:34:23 - deep_learning - INFO - [logger.py:103] - Starting training for SOL_1w_lstm
2025-05-27 01:34:37 - deep_learning - ERROR - [logger.py:111] - Error training SOL-1w-lstm: "['target'] not found in axis"
2025-05-27 01:34:37 - deep_learning - INFO - [logger.py:103] - Starting training for SOL_1w_gru
2025-05-27 01:34:51 - deep_learning - ERROR - [logger.py:111] - Error training SOL-1w-gru: "['target'] not found in axis"
2025-05-27 01:34:51 - deep_learning - INFO - [logger.py:103] - Starting training for SOL_1w_transformer
2025-05-27 01:35:05 - deep_learning - ERROR - [logger.py:111] - Error training SOL-1w-transformer: "['target'] not found in axis"
2025-05-27 12:40:22 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_lstm
2025-05-27 12:40:37 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-lstm: "['target'] not found in axis"
2025-05-27 12:40:37 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_gru
2025-05-27 12:40:51 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-gru: "['target'] not found in axis"
2025-05-27 12:40:51 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_transformer
2025-05-27 12:45:45 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_lstm
2025-05-27 12:46:00 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-lstm: "['target'] not found in axis"
2025-05-27 12:46:00 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_gru
2025-05-27 12:46:15 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-gru: "['target'] not found in axis"
2025-05-27 12:46:15 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_transformer
2025-05-27 12:46:29 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-transformer: "['target'] not found in axis"
2025-05-27 12:46:29 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1w_lstm
2025-05-27 12:46:45 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1w-lstm: "['target'] not found in axis"
2025-05-27 12:46:45 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1w_gru
2025-05-27 12:47:00 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1w-gru: "['target'] not found in axis"
2025-05-27 12:47:00 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1w_transformer
2025-05-27 12:47:14 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1w-transformer: "['target'] not found in axis"
2025-05-27 12:47:14 - deep_learning - INFO - [logger.py:103] - Starting training for SOL_1w_lstm
2025-05-27 12:47:29 - deep_learning - ERROR - [logger.py:111] - Error training SOL-1w-lstm: "['target'] not found in axis"
2025-05-27 12:47:29 - deep_learning - INFO - [logger.py:103] - Starting training for SOL_1w_gru
2025-05-27 12:47:43 - deep_learning - ERROR - [logger.py:111] - Error training SOL-1w-gru: "['target'] not found in axis"
2025-05-27 12:47:43 - deep_learning - INFO - [logger.py:103] - Starting training for SOL_1w_transformer
2025-05-27 12:47:58 - deep_learning - ERROR - [logger.py:111] - Error training SOL-1w-transformer: "['target'] not found in axis"
2025-05-27 12:53:02 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_lstm
2025-05-27 12:55:11 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_lstm
2025-05-27 13:00:07 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_lstm
2025-05-27 13:00:22 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-lstm: "['target'] not found in axis"
2025-05-27 13:00:22 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_gru
2025-05-27 13:00:36 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-gru: "['target'] not found in axis"
2025-05-27 13:00:36 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_transformer
2025-05-27 15:40:09 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_lstm
2025-05-27 15:40:57 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_lstm
2025-05-27 15:42:15 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_lstm
2025-05-27 15:42:21 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-lstm: "['target'] not found in axis"
2025-05-27 15:42:21 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_gru
2025-05-27 15:42:26 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-gru: "['target'] not found in axis"
2025-05-27 15:42:26 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_transformer
2025-05-27 15:42:32 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-transformer: "['target'] not found in axis"
2025-05-27 15:42:32 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1w_lstm
2025-05-27 15:42:37 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1w-lstm: "['target'] not found in axis"
2025-05-27 15:42:37 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1w_gru
2025-05-27 15:42:43 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1w-gru: "['target'] not found in axis"
2025-05-27 15:42:43 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1w_transformer
2025-05-27 15:44:11 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_lstm
2025-05-27 15:44:26 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-lstm: "['target'] not found in axis"
2025-05-27 15:44:26 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_gru
2025-05-27 15:44:41 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-gru: "['target'] not found in axis"
2025-05-27 15:44:41 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_transformer
2025-05-27 15:44:56 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-transformer: "['target'] not found in axis"
2025-05-27 15:44:56 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1w_lstm
2025-05-27 15:45:10 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1w-lstm: "['target'] not found in axis"
2025-05-27 15:45:10 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1w_gru
2025-05-27 15:45:25 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1w-gru: "['target'] not found in axis"
2025-05-27 15:45:25 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1w_transformer
2025-05-27 15:45:39 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1w-transformer: "['target'] not found in axis"
2025-05-27 15:45:39 - deep_learning - INFO - [logger.py:103] - Starting training for SOL_1w_lstm
2025-05-27 15:45:53 - deep_learning - ERROR - [logger.py:111] - Error training SOL-1w-lstm: "['target'] not found in axis"
2025-05-27 15:45:53 - deep_learning - INFO - [logger.py:103] - Starting training for SOL_1w_gru
2025-05-27 15:46:07 - deep_learning - ERROR - [logger.py:111] - Error training SOL-1w-gru: "['target'] not found in axis"
2025-05-27 15:46:07 - deep_learning - INFO - [logger.py:103] - Starting training for SOL_1w_transformer
2025-05-27 15:46:21 - deep_learning - ERROR - [logger.py:111] - Error training SOL-1w-transformer: "['target'] not found in axis"
2025-05-27 15:51:00 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_lstm
2025-05-27 15:51:14 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-lstm: "['target'] not found in axis"
2025-05-27 15:51:14 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_gru
2025-05-27 15:59:09 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_lstm
2025-05-27 15:59:23 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-lstm: "['target'] not found in axis"
2025-05-27 15:59:23 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_gru
2025-05-27 15:59:38 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-gru: "['target'] not found in axis"
2025-05-27 15:59:38 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_transformer
2025-05-27 15:59:52 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-transformer: "['target'] not found in axis"
2025-05-27 15:59:52 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1w_lstm
2025-05-27 16:00:06 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1w-lstm: "['target'] not found in axis"
2025-05-27 16:00:06 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1w_gru
2025-05-27 16:00:21 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1w-gru: "['target'] not found in axis"
2025-05-27 16:00:21 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1w_transformer
2025-05-27 16:00:35 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1w-transformer: "['target'] not found in axis"
2025-05-27 16:00:35 - deep_learning - INFO - [logger.py:103] - Starting training for SOL_1w_lstm
2025-05-27 16:00:49 - deep_learning - ERROR - [logger.py:111] - Error training SOL-1w-lstm: "['target'] not found in axis"
2025-05-27 16:00:49 - deep_learning - INFO - [logger.py:103] - Starting training for SOL_1w_gru
2025-05-27 16:01:03 - deep_learning - ERROR - [logger.py:111] - Error training SOL-1w-gru: "['target'] not found in axis"
2025-05-27 16:01:03 - deep_learning - INFO - [logger.py:103] - Starting training for SOL_1w_transformer
2025-05-27 16:01:17 - deep_learning - ERROR - [logger.py:111] - Error training SOL-1w-transformer: "['target'] not found in axis"
2025-05-27 16:10:05 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_lstm
2025-05-27 16:10:19 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-lstm: "['target'] not found in axis"
2025-05-27 16:10:19 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_gru
2025-05-27 16:10:34 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-gru: "['target'] not found in axis"
2025-05-27 16:10:34 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_transformer
2025-05-27 16:10:48 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-transformer: "['target'] not found in axis"
2025-05-27 16:10:48 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1w_lstm
2025-05-27 16:11:03 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1w-lstm: "['target'] not found in axis"
2025-05-27 16:11:03 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1w_gru
2025-05-27 16:11:18 - deep_learning - ERROR - [logger.py:111] - Error training ETH-1w-gru: "['target'] not found in axis"
2025-05-27 16:11:18 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1w_transformer
2025-05-27 16:11:27 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_lstm
2025-05-27 16:11:42 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-lstm: "['target'] not found in axis"
2025-05-27 16:11:42 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_gru
2025-05-27 16:11:56 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-gru: "['target'] not found in axis"
2025-05-27 16:11:56 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_transformer
2025-05-27 16:16:02 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_lstm
2025-05-27 16:16:17 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-lstm: "['target'] not found in axis"
2025-05-27 16:16:17 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_gru
2025-05-27 16:16:31 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-gru: "['target'] not found in axis"
2025-05-27 16:16:31 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_transformer
2025-05-27 16:16:46 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-transformer: "['target'] not found in axis"
2025-05-27 16:16:46 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_1w_lstm
2025-05-27 16:24:35 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_lstm
2025-05-27 17:00:20 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_lstm
2025-05-27 17:00:38 - deep_learning - ERROR - [logger.py:111] - Error training BTC-1w-lstm: "['target'] not found in axis"
2025-05-27 17:00:38 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_gru
2025-05-27 17:03:06 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1w_lstm
2025-05-27 17:28:51 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_1d_lstm
2025-05-27 17:30:56 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_lstm
2025-05-27 17:32:01 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-lstm: "['target'] not found in axis"
2025-05-27 17:32:01 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_gru
2025-05-27 17:32:58 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-gru: "['target'] not found in axis"
2025-05-27 17:32:58 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_transformer
2025-05-27 17:50:25 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_lstm
2025-05-27 17:51:20 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-lstm: "['target'] not found in axis"
2025-05-27 17:51:20 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_gru
2025-05-27 17:52:14 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-gru: "['target'] not found in axis"
2025-05-27 17:52:14 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_transformer
2025-05-27 17:53:07 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-transformer: "['target'] not found in axis"
2025-05-27 17:53:07 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_4h_lstm
2025-05-27 17:54:00 - deep_learning - ERROR - [logger.py:111] - Error training ETH-4h-lstm: "['target'] not found in axis"
2025-05-27 17:54:00 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_4h_gru
2025-05-27 18:18:09 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_lstm
2025-05-27 18:20:49 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-lstm: "['target'] not found in axis"
2025-05-27 18:20:49 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_gru
2025-05-27 18:40:51 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_lstm
2025-05-27 18:41:44 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-lstm: "['target'] not found in axis"
2025-05-27 18:41:44 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_gru
2025-05-27 18:42:34 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-gru: "['target'] not found in axis"
2025-05-27 18:42:34 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_transformer
2025-05-27 18:50:33 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_lstm
2025-05-27 18:51:23 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-lstm: "['target'] not found in axis"
2025-05-27 18:51:23 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_gru
2025-05-27 18:52:36 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_lstm
2025-05-27 18:53:28 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-lstm: "['target'] not found in axis"
2025-05-27 18:53:28 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_gru
2025-05-27 18:54:33 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_lstm
2025-05-27 18:55:22 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-lstm: "['target'] not found in axis"
2025-05-27 18:55:22 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_gru
2025-05-27 18:55:46 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_lstm
2025-05-27 18:56:27 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-lstm: "['target'] not found in axis"
2025-05-27 18:56:27 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_gru
2025-05-27 18:57:08 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-gru: "['target'] not found in axis"
2025-05-27 18:57:08 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_transformer
2025-05-27 18:57:49 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-transformer: "['target'] not found in axis"
2025-05-27 18:57:49 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_4h_lstm
2025-05-27 18:58:31 - deep_learning - ERROR - [logger.py:111] - Error training ETH-4h-lstm: "['target'] not found in axis"
2025-05-27 18:58:31 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_4h_gru
2025-05-27 18:59:12 - deep_learning - ERROR - [logger.py:111] - Error training ETH-4h-gru: "['target'] not found in axis"
2025-05-27 18:59:12 - deep_learning - INFO - [logger.py:103] - Starting training for ETH_4h_transformer
2025-05-27 19:05:58 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_lstm
2025-05-27 19:06:28 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-lstm: "['target'] not found in axis"
2025-05-27 19:06:28 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_gru
2025-05-27 19:06:56 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-gru: "['target'] not found in axis"
2025-05-27 19:06:56 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_transformer
2025-05-27 19:14:32 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_lstm
2025-05-27 19:15:13 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-lstm: "['target'] not found in axis"
2025-05-27 19:15:13 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_gru
2025-05-27 19:15:53 - deep_learning - ERROR - [logger.py:111] - Error training BTC-4h-gru: "['target'] not found in axis"
2025-05-27 19:15:53 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_transformer
2025-05-27 19:35:02 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_lstm
2025-05-27 22:47:01 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_lstm
2025-05-27 22:48:31 - deep_learning - INFO - [logger.py:103] - Starting training for BTC_4h_lstm
